modules/policy/engine/src/main/java/org/eclipse/sw360/antenna/policy/engine/RuleSet.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+import java.util.Collection;
+
+/**
+ * This is a collection of {@link Rule} objects within a common context. The configuration of the {@link PolicyEngine}
+ * in the {@link PolicyEngineConfigurator} allows to give several rule sets to set up the policies.
+ */
+public interface RuleSet {
+    String version();
```

Comment:
Why is the version a free-text string that makes it hard to compare what is newer vs. older? How about making this a monotonically increasing `int`, similar to Subversion revisions, for simple version management?

Reply:
I'll discuss that with Marcel, seems that I overlooked this comment of yours, sorry. Perhaps also your proposal makes sense. But I go with your comment, that we can change it later, at least when we  implement the first "official" Ruleset